print/ghostscript/Portfile

```diff
@@ -32,12 +32,13 @@ distfiles           ${distname}.tar.gz:source \
                     ${mappingresources_commit}.zip:misc
 
 patchfiles          patch-base_unix-dll.mak.diff \
```

Comment:
If this patchfile is no longer needed please remove it aswell.

Reply:
The makefile patches are still needed.